test/integ/test_put_get.py

```diff
@@ -646,3 +648,21 @@ def test_put_special_file_name(tmp_path, conn_cnx):
             ).fetchall()
             cur.execute(f"select $1, $2, $3 from  @{stage_name}")
             assert cur.fetchone() == ("1", "2", "3")
+
+
+@pytest.mark.skipolddriver
+def test_get_empty_file(tmp_path, conn_cnx):
+    test_file = tmp_path / "data.csv"
+    test_file.write_text("1,2,3\n")
+    stage_name = random_string(5, "test_get_empty_file_")
+    with conn_cnx() as cnx:
+        with cnx.cursor() as cur:
+            cur.execute(f"create temporary stage {stage_name}")
+            filename_in_put = str(test_file).replace("\\", "/")
+            cur.execute(
+                f"PUT 'file://{filename_in_put}' @{stage_name}",
+            ).fetchall()
+            empty_file = tmp_path / "foo.csv"
+            with pytest.raises(OperationalError):
+                cur.execute(f"GET @{stage_name}/{empty_file} file://{tmp_path}")
```

Comment:
I think here `empty_file` is an absolute path. I think what you meant is:
```suggestion
                cur.execute(f"GET @{stage_name}/foo.csv file://{tmp_path}")
```
I also think that the assert statement after isn't necessary. If we expect an exception then no file should be created